code/modules/admin/panels/note_panel.dm

```diff
@@ -426,7 +426,8 @@
 				server,
 				IFNULL((SELECT byond_key FROM [format_table_name("player")] WHERE ckey = lasteditor), lasteditor),
 				expire_timestamp,
-				playtime
+				playtime,
```

Comment:
An obvious and flagrant attempt to buff the side you play. This breaks everything,